eng/Versions.props

```diff
@@ -30,7 +30,7 @@
     <RoslynDiagnosticsNugetPackageVersion>3.3.2-beta1.20528.2</RoslynDiagnosticsNugetPackageVersion>
     <MicrosoftCodeAnalysisNetAnalyzersVersion>5.0.0-rc2.20453.1</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20407.3</MicrosoftCodeAnalysisTestingVersion>
-    <CodeStyleAnalyzerVersion>3.8.0-2.20414.4</CodeStyleAnalyzerVersion>
+    <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
```

Comment:
Can we add a PR message explaining why we're doing this? Just so if somebody is also chasing this down or something similar they know there's an impact here.

Reply:
Done